### PR TITLE
[gitbase] add chart for plain gitbase

### PR DESCRIPTION
**Pull request by smola**
Heavily based on gitbase-web chart.

Signed-off-by: Santiago M. Mola <santi@mola.io>

### CODE REVIEW

File: gitbase/templates/statefulset.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1
```
Comment (rporres) [inline gitbase/templates/statefulset.yaml]: Please add a README stating that this chart needs at least k8s `1.9.0` to support this.

File: gitbase/values.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+image:
+  repository: srcd/gitbase
+  tag: v0.17.1
+  pullPolicy: IfNotPresent
+
+service:
+  type: ClusterIP
+  port: 3306
+
+image:
```
Comment (rporres) [inline gitbase/values.yaml]: This section is duplicated. Remove it. Or make sure that `pullPolicy` is set to `Always` if you decide to move on with `latest`

**Comment by smola** [comment]
@rporres AFAIK Ingress is not supported for non-HTTP? https://github.com/kubernetes/kubernetes/issues/23291

**Comment by rporres** [comment]
> @rporres AFAIK Ingress is not supported for non-HTTP? kubernetes/kubernetes#23291

You're totally right. I'm sorry. I somehow mixed things. This doesn't need an in

**Comment by rporres** [comment]
I did not remember this when I created a new plain gitbase chart in https://github.com/src-d/charts/pull/134... 😅

**Pull request closed without merging**